www/src/components/docs/tabs.astro

```diff
@@ -0,0 +1,76 @@
+---
+const { tabName, slotOne, slotTwo } = Astro.props;
+---
+
+<style>
+  .tab-list {
+    display: flex;
+    cursor: pointer;
+  }
+
+  .tab {
+    padding: 0.5rem 1rem;
+    margin-right: 1rem;
+    font-weight: bold;
+    border: none;
+    background: none;
+    cursor: pointer;
+    outline: none;
+    transition: border-bottom 0.3s;
+  }
+
+  .tab.active {
+    border-bottom: 1px solid #4a3eeb;
+    color: #4a3eeb;
+  }
```

Comment:
this has quite low contrast in dark mode:

![CleanShot 2024-11-04 at 23 23 20@2x](https://github.com/user-attachments/assets/db46e652-3910-4563-98ae-dfb28ef66fb3)

---


maybe you can match the buttons from here: 

![CleanShot 2024-11-04 at 23 23 25@2x](https://github.com/user-attachments/assets/040208a2-3991-4f57-b16a-29186b813af7)